checkQC/default_config/config.yaml

```
@@ -323,6 +323,117 @@ novaseq_S4:
         warning: unknown
         error: 1500
 
+novaseqXPlus_1.5B:
```

```suggestion
novaseqxplus_1.5B:
```
To follow the format of the other instrument, it should be lower case.